### PR TITLE
[TS] LPS-155916 Cannot set Field Group with a nested Date Field repeatable

### PR DESCRIPTION
**Pull request by IstvanD**
Dear Team,

Could you please review this pull request?

Thank you and best regards,
István

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/fieldEditableReducer.es.js [hunk 1]
```diff
@@ -105,6 +105,8 @@ function updateFieldAffectedByActivatingRepeatable({
 }) {
 	if (
 		field.type === 'date' &&
+		field.validation &&
+		field.validation.parameter &&
```
Comment (miltonmc) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/fieldEditableReducer.es.js:109]: You can use [Optional chaining (?.)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to simplify even more:
```suggestion
		field.validation?.parameter &&
```

**Review by miltonmc — CHANGES_REQUESTED** [review]
Hi @IstvanD,

Thank you so much for this fix!

It looks great, but  if you could do the following tweak it will be perfect:

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/fieldEditableReducer.es.js (hunk 1 above)
Comment (IstvanD) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/fieldEditableReducer.es.js]: Hi @miltonmc thank you for your observations.
I force-pushed the recommended changes.

**Comment by IstvanD** [comment]
Dear Team,

I made the changes requested.
Please let me know if there are any updates on the review.

Thank you and best regards,
István

**Review by miltonmc — APPROVED** [review]
LGTM!

**Comment by miltonmc** [comment]
ci:forward

**Pull request closed without merging**